installation.rst

```diff
@@ -131,55 +132,26 @@ You can install these versions of the package by doing:
     release of VisPy so the Test PyPI package may be smaller than when
     the final package is released.
 
-Jupyter Notebook Extension
---------------------------
-
-If you would like to use the VisPy Jupyter Notebook Widget you must first
-install the ``ipywidgets`` library and enable the extension by doing:
-
-.. code-block:: console
-
-    pip install ipywidgets
-    jupyter nbextension enable --py widgetsnbextension
-    jupyter nbextension enable --py vispy
+Jupyter Notebook and Lab
+------------------------
 
-When using `virtualenv <https://virtualenv.pypa.io/en/stable/>`_ and working in
-an activated virtual environment, the ``--sys-prefix`` option may be required
-to enable the extension and keep the environment isolated (i.e.
-``jupyter nbextension enable --py widgetsnbextension --sys-prefix``).
-
-Note if you have an old version of the extension installed you mean need to
-manually delete it from `<python-prefix>/share/nbextensions/vispy`>
-
-If you are installing vispy from source in a "development mode" you may need
-to explicitly install the extension before enabling it (using the symlink
-option to link the extension to your active development environment):
-
-.. code-block:: console
+If you would like to use VisPy in a Jupyter Notebook and have the
+visualizations appear inline, we recommend using the "jupyter_rfb" backend.
+This backend depends on
+`jupyter_rfb library <https://jupyter-rfb.readthedocs.io/en/latest/>`_ which
+must be installed before your jupyter notebook or jupyter lab session is
+started.
 
-    jupyter nbextension install --symlink --py vispy --sys-prefix
-
-.. note::
-
-    The Jupyter-based backend and extension should be considered experimental
-    due to performance limitations; both by the WebGL standard and what is
-    currently implemented in vispy. Users should strongly consider these
-    limitations before using vispy for an operational WebGL application.
-
-JupyterLab
-----------
-
-To install the JupyterLab extension you need to install it explicitly with the
-following:
-
-.. code-block:: console
+Note that the 'jupyter_rfb' library uses the "remote" jupyter kernel
+(the server) to do the drawing of your visualization and then sends the
+results to the client (the browser). This means that performance of animations
+and user interactions (mouse and keyboard events) will differ depending on the
+connection speed and quality between server and client.
```

Comment:
Nitpick: I mentally placed comma at `on the connection speed, and quality between server and client` causing me to interpret it in a weird way. Maybe just "connection quality", or even just "connection".
```suggestion
connection between server and client.
```

Reply:
I went with connection quality. Good point. Thanks.